vercel.json

```diff
@@ -6,10 +6,10 @@
   },
   "redirects": [
     {"source": "/docs/:path*", "destination": "/:path*"},
-    {
-      "source": "/typescript/:path*",
-      "destination": "https://legacy-documentation-sdks.temporal.io/typescript/:path*"
-    },
+    {"source": "/typescript/:path*", "destination": "https://legacy-documentation-sdks.temporal.io/typescript/:path*"},
+    {"source": "/go/:path*", "destination": "https://legacy-documentation-sdks.temporal.io/go/:path*"},
+    {"source": "/java/:path*", "destination": "https://legacy-documentation-sdks.temporal.io/java/:path*"},
+    {"source": "/concepts/what-is:path*", "destination": "concepts/landing"},
```

Comment:
Why redirect to the landing page here?

Reply:
Would be nice to keep the old URLs working.

Reply:
@bergundy I will put in a redirect for each one before this merges.